### ISSUE TITLE
HTML file in the JS directory

### ISSUE THREAD

**Issue by humblewebdev** [danielpolk/Project-1#34]
https://github.com/danielpolk/Project-1/blob/51824412e0ef9f4c428ccb0f3adf67906b3a2b0b/js/APIs.html#L2

The JS directory should be just JS files.

**Comment by markpython86** [comment]
> [Project-1/js/APIs.html](https://github.com/danielpolk/Project-1/blob/51824412e0ef9f4c428ccb0f3adf67906b3a2b0b/js/APIs.html#L2)
> 
> Line 2 in [5182441](/danielpolk/Project-1/commit/51824412e0ef9f4c428ccb0f3adf67906b3a2b0b)
> 
>  <html> 
> The JS directory should be just JS files.

Yes, we were testing the API functionality separately and should've removed it.
